File: backend/test-login.py
import urllib.request
import json
import ssl

url = 'http://localhost:5000/api/v1/auth/login'
data = {
    "Username": "LaiqAhmed",
    "Password": '38403"Sargodha#Laiq@3939@Ahmed'
}
req = urllib.request.Request(url, data=json.dumps(data).encode('utf-8'))
req.add_header('Content-Type', 'application/json')

try:
    response = urllib.request.urlopen(req)
    print("Status:", response.status)
    print("Response:", response.read().decode('utf-8'))
except urllib.error.HTTPError as e:
    print("Status:", e.code)
    print("Response:", e.read().decode('utf-8'))
except Exception as e:
    print("Error:", str(e))

File: test-login.js
const fs = require('fs');

async function testUserFlow() {
  const username = "LaiqAhmed";
  const password = "38403\"Sargodha#Laiq@3939@Ahmed";
  
  console.log(`Testing with Username: ${username}`);
  console.log(`Testing with Password: ${password}`);

  try {
    const res = await fetch('http://localhost:5000/api/v1/auth/login', {
      method: 'POST',
      headers: { 'Content-Type': 'application/json' },
      body: JSON.stringify({ Username: username, Password: password })
    });
    
    const data = await res.text();
    console.log(`Status: ${res.status}`);
    console.log(`Response: ${data}`);
  } catch (err) {
    console.error("Error connecting to backend:", err.message);
  }
}

testUserFlow();
